app/build.gradle.kts

```diff
@@ -52,6 +52,9 @@ android {
             isIncludeAndroidResources = true
         }
     }
+    allprojects {
+        apply(from = "$rootDir/app/buildOptions.gradle")
```

Comment:
This can be:

`apply("$rootDir/app/buildOptions.gradle")`

Reply:
https://github.com/zhaonian/GHub/pull/20/files